receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -1 +1,268 @@
+  elasticsearch.node.network.connections:
+    description: Number of open tcp connections for internal cluster communication.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.http.connections:
+    description: Number of HTTP connections to the node.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.operations.completed:
+    description: Number of operations completed.
+    unit: "{operations}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [operation]
+  elasticsearch.node.operations.time:
+    description: Time spent on operations.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [operation]
+  elasticsearch.node.shards.size:
+    description: The total size of the shards assigned to this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.thread_pool.threads:
+    description: The number of threads in the thread pool.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [thread_state]
+  elasticsearch.node.thread_pool.tasks.queued:
+    description: The number of queued tasks in the thread pool.
+    unit: "{tasks}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [thread_pool_name]
+  elasticsearch.node.thread_pool.tasks.finished:
+    description: The number of tasks finished by the thread pool.
+    unit: "{tasks}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [thread_pool_name, task_state]
+  elasticsearch.node.documents:
+    description: The number of documents on the node.
+    unit: "{documents}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [document_state]
+  elasticsearch.node.open_files:
+    description: The number of open file descriptors held by the node.
+    unit: "{files}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  # these metrics are from /_cluster/health, and are cluster level metrics
+  elasticsearch.cluster.shards:
+    description: The number of shards in the cluster.
+    unit: "{shards}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [shard_type]
+  elasticsearch.cluster.data_nodes:
+    description: The number of data nodes in the cluster.
+    unit: "{nodes}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.cluster.nodes:
```

Comment:
Aside from "data nodes", what other types of nodes are there? It might make sense to combine these metrics, but only if aggregation across an attribute can produce the total.

Reply:
There are quite a few [node "roles"](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#node-roles), and a node may have multiple roles, so aggregation over them wouldn't necessarily produce the total, so I don't think combining the metrics should be done in this case.